tests/utils.py

```diff
@@ -3,16 +3,144 @@
 
 import asyncio
 import collections
+import sys
+from typing import Any
 import unittest
 from collections.abc import Mapping
+import concurrent.futures
 
 import plumpy
 from plumpy import persistence, process_states, processes, utils
-from plumpy.process_comms import  MessageBuilder
+from plumpy.exceptions import CoordinatorConnectionError
+from plumpy.message import MessageBuilder
+from plumpy.rmq import TaskRejected
+import shortuuid
 
 Snapshot = collections.namedtuple('Snapshot', ['state', 'bundle', 'outputs'])
 
 
+class MockCoordinator:
+    def __init__(self):
+        self._task_subscribers = {}
+        self._broadcast_subscribers = {}
+        self._rpc_subscribers = {}
+        self._closed = False
+
+    def is_closed(self) -> bool:
+        return self._closed
+
+    def close(self):
+        if self._closed:
+            return
+        self._closed = True
+        del self._task_subscribers
+        del self._broadcast_subscribers
+        del self._rpc_subscribers
```

Comment:
This is the Mocked in memory coordinator that in principle can work with sqlite in presto and make it able to submit the calculations.